buffer/src/main/java/io/netty5/buffer/adapt/AdaptivePoolingAllocator.java

```diff
@@ -140,7 +140,7 @@ protected Object initialValue() {
                         return NO_MAGAZINE;
                     }
                     Magazine mag = new Magazine(AdaptivePoolingAllocator.this, executor);
-                    if (FastThreadLocalThread.willCleanupFastThreadLocals(Thread.currentThread())) {
+                    if (!FastThreadLocalThread.currentThreadWillCleanupFastThreadLocals()) {
```

Comment:
This boolean should not be inverted.

Reply:
```suggestion
                    if (FastThreadLocalThread.currentThreadWillCleanupFastThreadLocals()) {
```

Reply:
yep, I think this check was flipped in 4.2, so my dumb copy failed :D